cmd/version/version.go

```diff
@@ -37,7 +47,25 @@ func NewCmdVersion(rootCmd *cobra.Command) *cobra.Command {
 		Use:   "version",
 		Short: "Shows openebs kubectl plugin's version",
 		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Println("Client Version: " + rootCmd.Version)
+			k, _ := client.NewK8sClient("")
+			nsMap, err := k.GetVersionMapOfComponents()
+
+			if err != nil {
+				fmt.Println("Client Version: " + rootCmd.Version)
+				fmt.Println("\nError getting Components Version")
+				return
+			}
+
+			header := []string{"Components", "Version"}
+			rows := [][]string{
+				{"Client", rootCmd.Version},
+				{"OpenEBS", getValidVersions(nsMap[util.OpenEBSProvisioner])},
```

Comment:
```suggestion
```

Please remove as this is getting deprecated and add it for `OpenEBS LVM LocalPV`

Reply:
So when we install openEBS in default mode following installation doc. 
Only these pods get created with component names like following:
```
maya-apiserver
admission-webhook
openebs-localpv-provisioner
ndm
ndm-operator
openebs-provisioner
openebs-snapshot-operator
```
So I since openebs-provisioner is gonna be deprecated, should I use `openebs-localpv-provisioner` to look for the version to reflect it as the version of openEBS itself?

Reply:
No you can leave them out for now

Reply:
Actually I have to use some pod version to show the version of OpenEBS component.
I was using `openebs-provisioner` pod to refer the volume but since it is deprecated, we can't use it. 
> Please remove as this is getting deprecated and add it for OpenEBS LVM LocalPV

Were you referring to use Persistent volumes to query version. Or I should just remove the openEBS version from the output table for the time being?

Reply:
> Actually I have to use some pod version to show the version of OpenEBS component.

That part is a bit unclear as of now, we'd have to observe how things evolve.

`OpenEBS` version might not make a lot of sense as with the newer helm chart, we can pick and choose different cas-type/storage-engine installations, moreover it's the storage engine version which matters more.

You can remove OpenEBS version for now, as it's a bit vague right now.

> Were you referring to use Persistent volumes to query version. Or I should just remove the openEBS version from the output table for the time being?

No, this is fine, just add printing option for `OpenEBS LVM LocalPV`.

Reply:
forgot adding LVM Local PV. Adding it.

Reply:
Done